process-manager/process-manager.js

```diff
@@ -20,7 +20,7 @@ ProcessManager.prototype.start = function(cb) {
   var pathToPm = require.resolve('strong-pm/bin/sl-pm');
   var args = [pathToPm, '--listen', '0', '--no-control'];
   var envOverrides = {
-    PORT: '3001'
+    PORT: '2999'
```

Comment:
is the designed behaviour to respect an explcit port assignment if one is available from the LB config.json file?